ocs_ci/utility/utils.py

```diff
@@ -3338,9 +3338,14 @@ def add_chrony_to_ocp_deployment():
             "machineconfiguration.openshift.io/role"
         ] = role
         chrony_template_obj["metadata"]["name"] = f"99-{role}-chrony-configuration"
-        chrony_template_obj["spec"]["config"]["ignition"][
-            "version"
-        ] = config.DEPLOYMENT["ignition_version"]
+        ignition_version = config.DEPLOYMENT["ignition_version"]
+        chrony_template_obj["spec"]["config"]["ignition"]["version"] = ignition_version
+
+        if Version.coerce(ignition_version) < Version.coerce("3.0"):
```

Comment:
Don't you need to also revert this change? https://github.com/red-hat-storage/ocs-ci/pull/4363/files